control-plane/roles/gardener-shoots/templates/shoot.j2

```diff
@@ -0,0 +1,131 @@
+apiVersion: core.gardener.cloud/v1beta1
+kind: Shoot
+metadata:
+  name: "{{ _config.name }}"
+  namespace: "{{ _config.namespace }}"
+  annotations:
+{% if 'managed_seed' in _config %}
+      shoot.gardener.cloud/managed-seed-api-server: "{{ _config.managed_seed.api_server | managed_seed_api_server_annotation() }}"
+{% endif %}
+      cluster.metal-stack.io/name: "{{ _config.name }}"
+      cluster.metal-stack.io/project: "{{ _config.project_id }}"
+      cluster.metal-stack.io/tenant: "{{ metal_control_plane_provider_tenant }}"
+      cluster.metal-stack.io/partition: "{{ _config.partition }}"
```

Comment:
I use these annotations and labels without understanding why metal-stack needs them:
```
  annotations:
    cluster.metal-stack.io/description: created by ansible
    cluster.metal-stack.io/name: "{{ shoot.name }}"
    cluster.metal-stack.io/owner: admin
    cluster.metal-stack.io/project: "{{ project_name }}"
    cluster.metal-stack.io/tenant: "{{ project_metal_stack_tenant_id }}"
    confirmation.gardener.cloud/deletion: "true"
  labels:
    cluster.metal-stack.io/name: "{{ shoot.name }}"
    cluster.metal-stack.io/partition: "{{ metal_partition_id }}"
    cluster.metal-stack.io/project: "{{ project_metal_stack_id }}"
```

Reply:
The confirmation deletion is dangerous. :sweat_smile: